server/src/main/java/org/elasticsearch/index/IndexSettings.java

```diff
@@ -248,20 +251,7 @@ public final class IndexSettings {
         Property.Dynamic,
         Property.IndexScope
     );
-    public static final TimeValue DEFAULT_REFRESH_INTERVAL = new TimeValue(1, TimeUnit.SECONDS);
-    public static final Setting<TimeValue> NODE_DEFAULT_REFRESH_INTERVAL_SETTING = Setting.timeSetting(
```

Comment:
I think this might be used internally and perhaps we should defer the cleanup here to avoid breaking internal clients, until it is removed from them?